src/lib/bootloader/grub2base.rb

```diff
@@ -56,7 +56,7 @@ def pmbr_setup(action, *devices)
     # Propose global options of bootloader
     def StandardGlobals
       # s390 do not have os_prober, see bnc#868909#c2
-      disable_os_prober = Arch.s390 || ProductFeatures.GetBooleanFeature("globals", "disable_os_prober")
+      disable_os_prober = (Arch.s390||Arch.ppc) || ProductFeatures.GetBooleanFeature("globals", "disable_os_prober")
```

Comment:
Does it make sense? For SLES global feature is set to false, so your change affected only SLED and opensuse ( opensuse is not for ppc ), so only SLED on ppc, do we release it there at all?